transcom/mymove
# B 17657 too can view updated sit params on sit resubmission

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/808731)

## Summary

When a service item is resubmitted, we want the TOO to be able to view the changed values. This allows those changed values to be displayed in a tooltip that shows the previous and current values for every changed item.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office application.
2. Login as a TOO.
3. Select a move with a resubmitted service item. I used move SITEXT for my testing.
4. Navigate to the Move task order page and scroll down to the Requested service items section.
5. Verify that a tooltip appears and that when hovering over the tooltip, you see the previous and current values for any details that changed during the resubmission.

### Frontend
